docs/API.md

```diff
@@ -2057,21 +2057,16 @@ Request body:
 
 #### GET /macaroon
 
-This endpoint returns a macaroon in JSON format that, when its third
-party caveats are discharged, will allow access to the charm store. No
+This endpoint returns a macaroon in JSON format that, when its third party caveats are discharged, will allow access to the charm store. No
 prior authorization is required.
 
 #### GET /delegatable-macaroon
 
 This endpoint returns a macaroon in JSON format that can be passed to
 third parties to allow them to access the charm store on the user's
-behalf.  A first party "is-entity" caveat may be added to restrict those
-parties so that they can only access a given charmstore entity with a
-specified id.
+behalf. The entity id must be added as a url encoded parameter and is added as a first party "is-entity" caveat to restrict the usability of the returned macaroon, so that third parties can only access a given charmstore entity with a the specified id.
```

Comment:
It would be nice if you could keep the length of these lines to around 80 chars.